src/output.c

```diff
@@ -768,3 +915,175 @@ void OutputNotifyFileRotation(void) {
         *(flag->flag) = 1;
     }
 }
+
+TmEcode OutputLoggerLog(ThreadVars *tv, Packet *p, void *thread_data)
+{
+    LoggerThreadStore *thread_store = (LoggerThreadStore *)thread_data;
+    RootLogger *logger = TAILQ_FIRST(&RootLoggers);
+    LoggerThreadStoreNode *thread_store_node = TAILQ_FIRST(thread_store);
+    while (logger && thread_store_node) {
+        if (logger->LogFunc != NULL) {
```

Comment:
Probably never, just became a bit idiomatic to check on funcptrs before calling them - this one is probably the exception.